content/oracles.mdx

```diff
@@ -0,0 +1,139 @@
+
+
+
+The bigger question that we have here is: what makes a good oracle? To answer that, we need to define the requirements a bit more.
+
+### Why Use Oracles?
+Oracles are the backbone of Smart Contracts
+
+Without them, many smart contracts lose their guarantees
+
+#### Oracles: Example
+
+
+**Online gambling market:**
+- 56,050,000$ annually
+- ~10,000,000 users
+- Roughly 1/20 adults will partake in some form of online gambling
+- 280% Growth since 2009
+
+<Resources
+    title="Gambling Data:" text="This market, with its significant growth and large market cap, has begun to be impacted by Blockchain."
+    resources={[
+        {link: "https://www.statista.com/statistics/270728/market-volume-of-online-gaming-worldwide/", text: "Statista.com"}
+    ]}
+/>
+
+It is typically nearly impossible to reliably determine and verify the results of many things that are used for gambling. For example, a game as simple as blackjack could be rigged in favor of the dealer to the tune of 1% gambling volume per day. With a smart contract, online gambling can be verifiably fair with transparency around any dealer biases present in games.
+
+However, many areas of online gambling are not as simple as blackjack. In sports betting, without an oracle, you cannot determine which team won a game; as a result, sports betting requires some sort of oracle. For a game like the Superbowl, **these contracts may begin to yield tens or hundreds of millions of dollars, creating a massive financial incentive for oracles to behave dishonorably, reporting data that would lead to their bets succeeding.**
+
+At this point, hopefully it is clear why we need oracles: without them, smart contracts are restricted to use cases that solely involve data that already exists on-chain, such as cryptocurrency. Once we have an oracle, we also need to consider if it is reliable enough for the use case at hand.
+
+##### Gambling Legality & Blockchain
+While traditional online gambling companies are tied to an established legal entity that could be sued in case of illegitimate behavior, smart contracts for gambling can be somewhat anonymous. Additionally, in places where gambling or online gambling may be illegal, gambling on the Blockchain is hard to prevent and prosecute.
+
+### Oracle Requirements
+Taking into consideration the why smart contracts and Blockchain networks might want to use oracles, here's how we would define these requirements:
+
+1. What degree of BFT does the oracle have?
+
+  * What would it cost an actor to push specific data through that oracle?
+
+2. What is the ultimate source of the data? (Company? Web feed? Sensors?)
+2. What is the cost of getting data?
+2. What is the delay in getting data?
+2.
```

Comment:
extra bullet?

Reply:
It's mostly a placeholder to remind me to be adding extra bullets, if I think of relevant stuff to add